tcpClientServer/src/client.cpp

```diff
@@ -0,0 +1,27 @@
+#include "tcpclient.h"
+#include <iostream>
+#include <cstring>
+
+bool argvContains(int argc, char *argv[], const char* string) {
+    for(int i = 0; i < argc; i++) {
+        if(std::strcmp(argv[i], string) == 0)
+            return true;
+    }
+    return false;
+}
+
+int main(int argc, char *argv[])
+{    
+    Q_UNUSED(argc);
+    Q_UNUSED(argv);
+
+    TcpClient client;
+    if(argvContains(argc, argv, "--only-init"))
+        client.setOnlyInit(true);
+    if(!client.init())
+    {
+        std::cerr << "Client initialization fail !!!" << '\n';
```

Comment:
Qui penso che potrebbe essere utile stampare il risultato di `QTcpSocket::errorString()`. Un utilizzatore di questo applicativo potrebbe non sapere cosa significhi `Client initialization fail`.
Penso potrebbe essere utile anche aggiungere una nota di ausilio del tipo `Was the tcp-server process running?`